src/common/config.h

```diff
@@ -136,6 +212,8 @@ struct md_config_t {
   // No metavariables will be returned (they will have already been expanded)
   int get_val(const char *key, char **buf, int len) const;
   int _get_val(const char *key, char **buf, int len) const;
+  config_value_t get_val(const char *key) const;
```

Comment:
Since we can't catch such mistakes at compile time, I'd rather assert the failure instead of having sections of code locally re-implement default values should the ```get_val``` fail. If I was attempting to retrieve the "rbd_default_features" and it was misspelled in code or renamed in the config, I'd want my code to fail early.